cpp/cmake/thirdparty/patches/nvbench_override.json

```diff
@@ -7,11 +7,6 @@
           "file" : "${current_json_dir}/nvbench_global_setup.diff",
           "issue" : "Fix add support for global setup to initialize RMM in nvbench [https://github.com/NVIDIA/nvbench/pull/123]",
           "fixed_in" : ""
-        },
```

Comment:
Note for posterity, this functionality is now upstreamed to nvbench: https://github.com/NVIDIA/nvbench/pull/140/files#diff-23659490fcf438ba3a51b15b456f5f73e7269da28327e4ff2fc8bc232dbc4860